contractsdk/cpp/XRC/xrc01/src/main/xrc01_e1.cc

```diff
@@ -18,6 +18,10 @@ bool safe_stoull(const std::string* in, uint64_t* out) {
     if (!in) {
         return false;
     }
+    if (in->substr(0,1) == "-") {
```

Comment:
It's not best way to check a number is signed or not. I cannot see any trim operation on input string, so " -100" could pass this check but result in 18446744073709551516 using stoull.

Reply:
This function will return `false` and the method will return.

Reply:
Thanks, this problem has been fixed.